layouts/shortcodes/api/cloud/v2-prefer.html

```diff
@@ -0,0 +1 @@
+For new workloads, use the [InfluxDB v2 API `/api/v2/write` and `/api/v2/query` endpoints](/influxdb/cloud-dedicated/primers/api/v2/).
```

Comment:
The v2 query endpoint will not work. It's a Flux endpoint and dedicated doesn't support Flux.

```suggestion
For new workloads, use the [InfluxDB v2 API `/api/v2/write` endpoint](/influxdb/cloud-dedicated/primers/api/v2/) for writing data and a Flight SQL client for querying data.
```

Reply:
> The v2 query endpoint will not work. It's a Flux endpoint and dedicated doesn't support Flux.

@sanderson Oops. Lapse of reason.